roles/ceph-validate/tasks/check_devices.yml

```diff
@@ -29,10 +30,11 @@
   when:
     - osd_scenario == 'non-collocated'
     - item.failed
+    - dedicated_devices is defined
 
-- name: fail if dedicated_device is not the size length as devices
+- name: fail if number of dedicated_devices is not equal to number of devices
   fail:
-    msg: "dedicated_device lengtth must be identical to devices's length"
+    msg: "Number of dedicated_devices must be equal to number of devices"
   when:
     - osd_scenario == 'non-collocated'
-    - dedicated_devices|length != devices|length
\ No newline at end of file
+    - dedicated_devices|length != devices|length
```

Comment:
I believe this would fail if ``dedicated_devices`` is not defined.

The ``lvm_batch`` test should cover that I think. It failed in the pipeline tests before getting to this point. I've started it again.